bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/JSONParser.java

```diff
@@ -53,6 +54,13 @@ public StateMachine initialValue() {
         }
     };
 
+    /**
+     * Represents the modes which process numeric values.
+     */
+    public enum NonStringValueProcessingMode {
+        FROM_JSON_STRING, FROM_JSON_FLOAT_STRING, FROM_JSON_DECIMAL_STRING
```

Comment:
Shall we document these fields since these modes aren't quite intuitive when it comes to the JSONParser?

Reply:
Added.